ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -34,14 +34,22 @@ public sealed class ObservableAsPropertyHelper<T> : IHandleObservableErrors, IDi
         /// changes, typically this will call the ViewModel's
         /// RaisePropertyChanged method.</param>
         /// <param name="initialValue">The initial value of the property.</param>
+        /// <param name="subscribeImmediately">
+        /// A value indicating, whether the <see cref="ObservableAsPropertyHelper{T}"/> 
+        /// should immediately subscribe to the source <paramref name="observable"/>, or not.
+        /// 
+        /// If this parameter is <c>false</c>, the <see cref="ObservableAsPropertyHelper{T}"/> 
+        /// subscribes to the <paramref name="observable"/> when the first <see cref="Value"/> is requested.
+        /// </param>
         /// <param name="scheduler">The scheduler that the notifications will be
         /// provided on - this should normally be a Dispatcher-based scheduler
         /// (and is by default)</param>
         public ObservableAsPropertyHelper(
             IObservable<T> observable, 
             Action<T> onChanged, 
-            T initialValue = default(T), 
-            IScheduler scheduler = null) : this(observable, onChanged, null, initialValue, scheduler) {}
+            T initialValue = default(T),
+            bool subscribeImmediately = false,
```

Comment:
Inserting `subscribeImmediately` here, rather than adding at the end does mean it'll break any code expecting this position to be a scheduler. I know breaking changes are acceptable in a major version bump, but is passing true here going to be more likely than wanting to specify a scheduler?

It might be worth adding this ctor as an overload, so we keep the scheduler at position 4 as well as adding this new flag.

The same comments apply to `ToProperty(..)`